docs/zk_security.md

```diff
@@ -0,0 +1,56 @@
+# ZooKeeper security
+
+To prevent accidental or malicious tampering with the data that Helios stores in ZooKeeper Helios
```

Comment:
grammar nit: should be a comma in `... with the data that Helios stores in Zookeeper, Helios...`